frontend/src/concepts/pipelines/content/compareRuns/CompareRunsRunList.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { ExpandableSection } from '@patternfly/react-core';
+import { TableVariant } from '@patternfly/react-table';
+import { PipelineRunKFv2 } from '~/concepts/pipelines/kfTypes';
+import { PipelinesFilter } from '~/concepts/pipelines/types';
+import { Table, useCheckboxTable } from '~/components/table';
+import CompareRunTableRow from '~/concepts/pipelines/content/compareRuns/CompareRunTableRow';
+import { compareRunColumns } from '~/concepts/pipelines/content/compareRuns/columns';
+import DashboardEmptyTableView from '~/concepts/dashboard/DashboardEmptyTableView';
+import usePipelineFilter, {
+  FilterOptions,
+  getDataValue,
+} from '~/concepts/pipelines/content/tables/usePipelineFilter';
+import CompareRunTableToolbar from '~/concepts/pipelines/content/compareRuns/CompareRunTableToolbar';
+
+type CompareRunsRunListProps = {
+  runs: PipelineRunKFv2[];
+  loading: boolean;
```

Comment:
becuase we are using context, you could just call the context here instead of passing it through, there wont be any extra api cost bc everything is shared